src/utils/setup/fetch-build-time-media.js

```diff
@@ -1,27 +1,48 @@
 import { initStitch } from './init-stitch';
 import { STITCH_AUTH_APP_ID } from '../../constants';
-import { parsePodcasts } from '../parse-podcasts';
 import { transformTwitchResponse } from '../transform-twitch-response';
 import { transformYoutubeResponse } from '../transform-youtube-response';
+import { transformPodcastResponse } from '../transform-podcast-response';
 import memoizerific from 'memoizerific';
 
 const fetchMedia = async () => {
     const client = await initStitch(STITCH_AUTH_APP_ID);
-    const [youtubeVideos, twitchVideos, lybsinPodcasts] = await Promise.all([
-        client.callFunction('fetchYoutubeData', []),
-        client.callFunction('fetchMDBTwitchVideos', []),
-        client.callFunction('fetchLybsinPodcasts', []),
+    const [
+        strapiYoutubeVideos,
+        strapiTwitchVideos,
+        strapiPodcasts,
+        strapiPodcastSeries,
+    ] = await Promise.all([
+        client.callFunction('fetchYoutubeDataFromStrapi', []),
+        client.callFunction('fetchTwitchDataFromStrapi', []),
+        client.callFunction('fetchPodcastsDataFromStrapi', []),
+        client.callFunction('fetchPodcastsSeriesDataFromStrapi', []),
     ]);
-    const allTwitchVideos = twitchVideos.data.map(transformTwitchResponse);
+    const allYoutubeVideos = strapiYoutubeVideos.map(transformYoutubeResponse);
+    const allTwitchVideos = strapiTwitchVideos.map(transformTwitchResponse);
+    const allPodcasts = strapiPodcasts.map(transformPodcastResponse);
     return {
-        allVideos: [
-            youtubeVideos.items.map(transformYoutubeResponse),
-            allTwitchVideos,
-        ].flat(),
-        allPodcasts: parsePodcasts(lybsinPodcasts),
+        allVideos: [allYoutubeVideos, allTwitchVideos].flat(),
+        allPodcasts: allPodcasts,
         fallbackTwitchVideo: allTwitchVideos[0],
+        podcastSeries: getPodcastsSeriesMapping(strapiPodcastSeries),
     };
 };
+
+const getPodcastsSeriesMapping = podcastSeries => {
+    const mapping = podcastSeries.map(podcastSeriesItem =>
+        getPodcastToSeriesMapping(podcastSeriesItem)
+    );
+    return mapping;
+};
+
+const getPodcastToSeriesMapping = podcastSeriesItem => {
+    const podcastItems = podcastSeriesItem.seriesEntry.map(entry => {
+        return { slug: entry.podcast.slug, title: entry.podcast.title };
+    });
+    return { articles: podcastItems, title: podcastSeriesItem.title };
+};
```

Comment:
In the future, will we want to link articles as well or videos? May need to generalize